src/Lunr/Ticks/PHPMailer/PHPMailer.php

```diff
@@ -0,0 +1,219 @@
+     */
+    protected readonly TracingControllerInterface&TracingInfoInterface $tracingController;
+
+    /**
+     * Profiling level
+     * @var AnalyticsDetailLevel
+     */
+    protected AnalyticsDetailLevel $analyticsDetailLevel;
+
+    /**
+     * Start time of the mail sending
+     * @var float
+     */
+    protected float $startTimestamp;
+
+    /**
+     * Constructor.
+     *
+     * @param bool|null $exceptions Should we throw external exceptions?
+     */
+    public function __construct(?bool $exceptions = NULL)
+    {
+        $this->analyticsDetailLevel = AnalyticsDetailLevel::None;
+
+        parent::__construct($exceptions);
+
+        /**
+         * The type of action_function is defined as string but it should be callable, so we ignore the phpstan warning
+         * @phpstan-ignore assign.propertyType
+         */
+        $this->action_function = [ $this, 'afterSending' ]; // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+    }
+
+    /**
+     * Destructor.
+     */
+    public function __destruct()
+    {
+        unset($this->startTimestamp);
+        unset($this->analyticsDetailLevel);
+
+        parent::__destruct();
+    }
+
+    /**
+     * Enable PHPMailer request analytics.
+     *
+     * @param EventLoggerInterface                            $eventLogger Instance of an event logger
+     * @param TracingControllerInterface&TracingInfoInterface $controller  Instance of a tracing controller
+     * @param AnalyticsDetailLevel                            $level       Analytics detail level (defaults to Info)
+     *
+     * @return void
+     */
+    public function enableAnalytics(
+        EventLoggerInterface $eventLogger,
+        TracingControllerInterface&TracingInfoInterface $controller,
+        AnalyticsDetailLevel $level = AnalyticsDetailLevel::Info,
+    ): void
+    {
+        $this->eventLogger          = $eventLogger;
+        $this->tracingController    = $controller;
+        $this->analyticsDetailLevel = $level;
+    }
+
+    /**
+     * Create a message and send it.
+     * Uses the sending method specified by $Mailer.
+     *
+     * @return bool false on error - See the ErrorInfo property for details of the error
+     */
+    public function send(): bool
+    {
+        if ($this->analyticsDetailLevel === AnalyticsDetailLevel::None)
+        {
+            return parent::send();
+        }
+
+        $this->startTimestamp = microtime(TRUE);
+
+        $this->tracingController->startChildSpan();
+
+        return parent::send();
+    }
+
+    /**
+     * Callback after each mail send
+     *
+     * @param bool                                          $isSent  Result of the send action
+     * @param array{0: string,1?: string}                   $to      Email addresses of the recipients
+     * @param array<array{0: string,1?: string}>            $cc      Cc email addresses
+     * @param array<array{0: string,1?: string}>            $bcc     Bcc email addresses
```

Comment:
You're not doing anything with these, while they are arguably the most important ones to figure out whether sending to a specific email worked or not.